ci-operator/config/openshift/file-integrity-operator/openshift-file-integrity-operator-master.yaml

```diff
@@ -107,26 +113,32 @@ tests:
 - as: e2e-bundle-aws
   steps:
     cluster_profile: aws
-    dependencies:
-      OO_INDEX: ci-index-file-integrity-operator-bundle
-    env:
-      OO_CHANNEL: alpha
-      OO_INSTALL_NAMESPACE: file-integrity
-      OO_PACKAGE: file-integrity-operator
-      OO_TARGET_NAMESPACES: '!install'
     test:
+    - as: install
+      cli: latest
+      commands: |
+        oc create namespace openshift-file-integrity
+        operator-sdk run bundle --security-context-config restricted -n openshift-file-integrity "$OO_BUNDLE"
```

Comment:
I wonder with step would we need to change to Make file for e2e test

Reply:
So that the e2e suite doesn't install the operator?

Reply:
I believe `TEST_BUNDLE_INSTALL=1` bypasses the install procedure in the e2e suite.